config/services.yml

```diff
@@ -143,3 +143,27 @@
   csproj_path: 'GetIntoTeachingApi/GetIntoTeachingApi.csproj'
   docsets:
     - path: 'docs'
+- name: "Teacher pay calculator"
+  repo_name: "DFE-Digital/teacher-pay-calculator"
+  language: 'ruby'
+  docsets: []
+- name: "Refer a teacher for serious misconduct"
```

Comment:
This one is already on 120 with a slightly different name. I think the existing entry is correct.